tests/acceptance/integration-test.js

```diff
@@ -55,3 +58,21 @@ test('sticky messages are not removed automatically', function(assert) {
     assert.equal(find('.alert.alert-danger p').text(), 'You went offline');
   });
 });
+test('sticky messages are still removed when clicked', function(assert) {
+  assert.expect(5);
```

Comment:
Typically you don't need to `assert.expect` -- unless you're doing something async that the ember async helpers don't already handle. I'd remove this and see if the test still passes